src/main/java/chatbot/command/Command.java

```diff
@@ -0,0 +1,23 @@
+package chatbot.command;
+
+import chatbot.ChatbotException;
+import chatbot.Ui;
+import chatbot.task.TaskManager;
+
+
+/**
+ * Abstract base class for different types of commands that can be executed by the chatbot.
+ */
+public abstract class Command {
```

Comment:
Your use of an inheritable Abstract Command class to facilitate polymorphic behaviour and substitution is fantastic!